src/Umbraco.Web/Models/ContentEditing/CodeFileDisplay.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.Serialization;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Umbraco.Web.Models.ContentEditing
+{
+    [DataContract(Name = "scriptFile", Namespace = "")]
+    public class CodeFileDisplay
+    {
+        [DataMember(Name = "virtualPath")]
+        public string VirtualPath { get; set; }
```

Comment:
This model needs data annotations for validations applied - for example, VirtualPath cannot be null, it probably can be validated by a regex statement - the controller would then need to validate that it's an acceptable path